<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="user1" failures="1" tests="1" name="runner.Rerunrunner" time="7.469" errors="0" timestamp="2025-04-10T12:43:31 IST" skipped="0">
  <testcase classname="runner.Rerunrunner" name="runScenario" time="7.469">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at stepdef.searchbookstepdef.should_display_the_crime_thriller_books(searchbookstepdef.java:47)
at âœ½.should display the Crime & Thriller books(file:///C:/workspace/prasanthautomation/src/test/resources/features/BooksearchTest.feature:12)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- runner.Rerunrunner -->
